source/aks/patching-artifactory.html.md.erb

```diff
@@ -45,28 +72,97 @@ If any issues arise during the update process, troubleshooting steps are provide
+  • Submit your PR and ensure it gets reviewed and merged.
   
-  • Repeat [step 2](#2-check-new-version-on-cft-ptlsbox) checking the new version on CFT PTL ensuring the pods are healthy.
+  • Repeat [step 2](#2-check-new-version-on-cft-ptlsbox) for **'CFT PTL cluster'** checking the Helm release is on the new version and ensuring the pods are healthy.
 
-### Related Links
+## Related Links
   
   [GitHub repo](https://github.com/hmcts/cnp-flux-config)
   
-  [Artifactory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/hmcts) **Note** you will need to be connected to F5 VPN to access this link.
+  [Artifactory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/hmcts) **Note:** you will need to be connected to F5 VPN to access this link.
 
   [Artifactory Release Information](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes)
+
+## Troubleshooting
+
+### Missing required service
+
+```json
+[jfrou] [WARN] [217db729f7674ef0] [local_topology_helper.go:68] [main] [] - Missing required services: [jfmd]
+```
+
+You will see this error upon startup of artifactory as all services startup, if they do not clear after 60 - 90 seconds investigate further. This often occurs when one of the containers is not healthy, usually the router service. These errors most likely point to Master / Join key issues as the router cannot establish connection with the internal DB.
+
+### Join / Master key errors
+
+```json
+[jfrou] Cluster join: Access Service ping failed, will retry. Error: cluster join: error from service registry on ping:
+```
+
+Join key issues are found before master key issues arise, these are shown upon startup as the internal database boots and usually disappear after 30 seconds. If `join-key missing` is still showing investigate if the key has been picked up by artifactory correctly.
+
+```json
+[jfmd ] [ERROR] - Failed resolving master key: failed resolving 'shared.security.masterKey' key; file does not exist: /opt/jfrog/artifactory/var/etc/security/master.key
+```
+
+Master key is also used for the database, for internal communication purposes. If `master-key` issues are still showing ensure artifactory has these in the containers.
+
+#### Check keys in container
+
+```command
+kubectl exec -it artifactory-oss-0 -n artifactory -c router -- /bin/sh
+```
+
+Once inside the container cd to this directory, list the files and ensure `join.key` and `master.key` are there.
+
+```command
+cd var/etc/security/
+```
+
+```command
+ls -la
+```
+
+#### How to fix
+
+If the key issues are appearing restart the artifactory service by deleting the `pvc`, `statefulset` and the `pod` of artifactory namespace. If the service is still not rebooting successfully, then a full rebuild of the secret keys is most likely needed. Please reach out to senior member of the team.
```

Comment:
I think we could doc this instead of reach out to member of team
it's just random generated string, update secrets in flux, blow away pvc, pods and helm release and let flux re-install

Reply:
I also think it's maybe a given that if people are still stuck after that they'll reach out